lib/node_modules/@stdlib/blas/ext/base/gfill-by/benchmark/benchmark.js

```diff
@@ -21,16 +21,26 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var uniform = require( '@stdlib/random/base/uniform' ).factory;
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
```

Comment:
I'd stick with `uniform` here, as it is lighter weight than `discrete-uniform`. And instead of `rand` below, you can stick with using the `factory` method, as done previously. In general, this benchmark could be left as is and be fine.